src/TransCreator/PageBanner.jsx

```diff
@@ -1,19 +1,20 @@
 import React from 'react';
 import {
-  View, Text, Platform,
+  View, Text,
 } from 'react-native';
-import { Input, Icon } from 'react-native-elements';
+import { Icon } from 'react-native-elements';
 import { LinearGradient } from 'expo-linear-gradient';
 
 import PropTypes from 'prop-types';
 import themeStyle from '../Common/themeStyle';
 import themeColor from '../Common/Color';
 import styles from './Style';
-import numDecorator from './numericInputDecorator';
 
 export default function PageBanner({
-  transLabel, transType, transAmount, setTransAmount,
+  transLabel, transType, transAmount, transExpression,
 }) {
+  let displayAmount = transAmount === '' ? '0.00' : transAmount;
+  displayAmount = transType === 'Expense' ? `-$${displayAmount}` : `+$${displayAmount}`;
```

Comment:
At `utils.js` in `Stats` folder, it has a function to display expense or income according to type, may use it

Reply:
fixed